homeassistant/components/solarlog/strings.json

```
@@ -58,6 +58,15 @@
   },
   "entity": {
     "sensor": {
+      "charge_level": {
+        "name": "Charge level"
+      },
+      "charging_power": {
+        "name": "Charging battery"
+      },
+      "discharging_power": {
+        "name": "Discharging battery"
```

The friendly names should match the key names here, otherwise this will not result in proper translations.

```suggestion
      "charging_power": {
        "name": "Charging power"
      },
      "discharging_power": {
        "name": "Discharging power"
```

"Charging" / "Discharging" make inherently clear that we're talking about battery measurements.